Automattic/wp-calypso
# Login: Make it possible to send 2FA recovery code via sms

Requires D5495-code.

This PR updates the "Send recovery code via text" links on `WaitingNotificationApproval` and `VerificationCodeForm` to send a request to the API to actually send the recovery text.

**Testing**
- Apply D5495-code.

*Code verification screen*
- Visit http://calypso.localhost:3000/login
- Submit a username/password for an account that has SMS as its primary or secondary auth method.
- Click `Send recovery code via text`.
- Assert that you receive a text message on the phone number configured for the account.

*Push notification verification screen*
- Visit http://calypso.localhost:3000/login
- Submit a username/password for an account that has push as its primary auth method.
- Click `A recovery code via text`.
- Assert that you receive a text message on the phone number configured for the account.

*Throttling*
- Assert that an error notice is displayed when clicking either recovery code link twice in under a minute.
- Assert that waiting a minute then clicking the code results in an SMS being sent properly.
